x/concentrated-liquidity/incentives_test.go

```diff
@@ -2383,6 +2382,9 @@ func (s *KeeperTestSuite) TestQueryAndCollectIncentives() {
 				clKeeper := s.App.ConcentratedLiquidityKeeper
 				ctx := s.Ctx
 
+				// Set current tick according to test case
+				validPool.SetCurrentTick(tc.currentTick)
```

Comment:
Out of curiosity, what we the reasoning behind moving the current tick setting up here? It shouldn't actually make a difference since this doesn't go into effect until `SetPool` is run, no?

Reply:
For this specific test case, we can set this back. Done here https://github.com/osmosis-labs/osmosis/pull/5495/commits/9b2f7dd4beb6af5038df4565e5ff1abd376225bf